### ISSUE TITLE
Large-v3 CIF model

### ISSUE THREAD

**Issue by LAnCeBabY** [backspacetg/simul_whisper#3]
Hello！Your work is very good but I can‘t find the large-v3 version of cif model. How can I get / train a large-v3 version cif model?

**Comment by backspacetg** [comment]
Hi! You can use a single linear layer or CNN+ linear layer as the CIF model, and the input is the features extracted from the Whisper encoder, the label is the number of words in the labeled text, and the loss is RMSE. Code implementation and training methods for CIF models can be found [here](https://github.com/dqqcasia/mosst/blob/26eaaa61557aecae9e83410e7fddb90cffe2a704/fairseq/models/speech_to_text/convtransformer_wav2vec_cif.py#L307).

**Comment by twmht** [comment]
@backspacetg 

how many training data you use to train the CIF model

**Comment by backspacetg** [comment]
We used the 100-hour Librispeech train-clean data
